<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="W11VM-120" failures="1" tests="13" name="Default test" time="16.754" errors="0" timestamp="2024-11-21T11:29:46 MEZ">
  <testcase classname="executors.SauceDemoTest" name="test1" time="0.608"/>
  <testcase classname="executors.SauceDemoTest" name="test2" time="0.269"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="0.588"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="0.44"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="0.498"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="0.47"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="5.537"/>
  <testcase classname="executors.SauceDemoTest" name="test3" time="0.425"/>
  <testcase classname="executors.SauceDemoTest" name="test4" time="0.623"/>
  <testcase classname="executors.SauceDemoTest" name="test5" time="0.938"/>
  <testcase classname="executors.SauceDemoTest" name="test6" time="1.139"/>
  <testcase classname="executors.SauceDemoTest" name="test7" time="1.253"/>
  <testcase classname="executors.SauceDemoTest" name="test8" time="0.0">
    <failure type="org.testng.TestNGException" message="
Some DataProvider public java.lang.String[][] managers.DataManager.dataProvider(java.lang.String) parameters unresolved:  at 0 type class java.lang.String
">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public java.lang.String[][] managers.DataManager.dataProvider(java.lang.String) parameters unresolved:  at 0 type class java.lang.String

at org.testng.internal.invokers.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:237)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:156)
at org.testng.internal.Parameters.handleParameters(Parameters.java:812)
at org.testng.internal.Parameters.handleParameters(Parameters.java:737)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:915)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- test8 -->
</testsuite> <!-- Default test -->
